docker-compose/.amd64.env

```diff
@@ -0,0 +1,3 @@
+CONFIG_DIR=./config
+KUKSA_URL=https://kuksaval.northeurope.cloudapp.azure.com/job/kuksaval-upstream/job/master/4/artifact/artifacts/kuksa-val-863db65-amd64.tar.xz
+KUKSA_VAL_IMG=amd64/kuksa-val:863db65
```

Comment:
This is sample environment for AMD builds.
There is another for ARM builds.

you should execute ./run.sh <env file>

run.properties defines whether kuksa.val is used: USE_KUKSA_VAL=1
because URLs are ARCH specific, they are described in repective sample environments.

.env is not  touched on purpose to prevent merge conflicts for user's default environment

Reply:
There is no .env file anymore, which defaults to AMD64 architecture